src/Temporalio/Client/TemporalClientOptions.cs

```diff
@@ -0,0 +1,32 @@
+namespace Temporalio.Client
+{
+    /// <summary>
+    /// Options for a <see cref="TemporalClient" />.
+    /// </summary>
+    public class TemporalClientOptions
```

Comment:
```suggestion
    public class TemporalClientOptions : ICloneable
```

I implemented the method but not the interface

Reply:
Looks like I missed it on `RpcOptions`, `Testing.TemporaliteOptions`, etc...

Reply:
Fixed